### PR TITLE
YJDH-115 | Application attachments

### PR DESCRIPTION
**Pull request by klemmari1**
## Description :sparkles:

- Add Azure blob storage configuration.
- Add file fields for payslip and employment contract.

## Issues :bug:

[YJDH-115](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-115)

### CODE REVIEW

File: backend/kesaseteli/requirements.in [hunk 1]
```diff
@@ -6,6 +6,7 @@ django-environ
 django-extensions
 djangorestframework
 django-simple-history
+django-storages[azure]
```
Comment (haavikko) [inline backend/kesaseteli/requirements.in:9]: This is the same library I was planning to use in Helsinki benefit :-)

File: .env.kesaseteli.example [hunk 2]
```diff
@@ -24,5 +24,10 @@ ADFS_TENANT_ID=
 ADFS_LOGIN_REDIRECT_URL=
 ADFS_LOGIN_REDIRECT_URL_FAILURE=
 
+# Django-storages
+AZURE_ACCOUNT_NAME=
+AZURE_ACCOUNT_KEY=
+AZURE_CONTAINER=
```
Comment (haavikko) [inline .env.kesaseteli.example:30]: Add something to e.g. README, how to determine these values, and how to run locally with file system storage?

**Review by haavikko — APPROVED** [review]
Could use some documentation about config variables, other than that, looks good to me!

File: .env.kesaseteli.example (hunk 2 above)
Comment (klemmari1) [reply .env.kesaseteli.example:30]: I put this in the settings so locally this should not require any changes:
```
DEFAULT_FILE_STORAGE=(str, "django.core.files.storage.FileSystemStorage"),
```

I'll add a mention to the README about the AzureStorage.

**Pull request merged**